### DISCUSSION TITLE
Missing @fluentui/react-datepicker-compat

### DISCUSSION THREAD

**Discussion by m-a-zhou** [microsoft/fluentui#27469] [Q&A] [answered]
Hello,
I updated my SPFx solution to the newest version 9.18.7 to use the DatePicker Compat. But I cannot find the component
import { DatePicker } from '@fluentui/react-datepicker-compat';
The example https://codesandbox.io/s/7tor1y?file=/example.tsx shows "Cannot find module: @fluentui/react-datepicker-compat too
Can someone assist me to find the component?

**Comment by khmakoto** [comment]
Seems like the package is not being published to npm feed right now, I see it has `private: true` in its `package.json`

@sopranopillow, should we make it public since the package is now on the docsite?

**Reply by m-a-zhou** [reply]
Would be great! I'm developing an SPFx dynamic form webpart based on the controls and I need the datepicker. Thanks.

**Reply by sopranopillow** [reply]
I'll be making it public soon. I'll close this question once the package has been published.

**Reply by sopranopillow** [reply, marked as answer]
Hey, this PR resolves the issue: https://github.com/microsoft/fluentui/pull/27524
